src/components/common/footer/getstarted.js

```diff
@@ -14,6 +14,14 @@ const GetStarted = () => {
     window.open("https://github.com/loophole/cli/releases/latest");
   };
 
+  const subTitle = <Subtitle>
+    <b>
+      {" "}
```

Comment:
Accident?

Reply:
No, i thought why are we using bold one time and not the other, and if it's completely the same, it could be like this.

Reply:
I meant `{" "}`. Do we need this space in `<b>`?

Reply:
It was there before, I guess we don't need it